src/shared/awsContext.ts

```diff
@@ -6,12 +6,9 @@
 import * as vscode from 'vscode'
 
 // Carries the current context data on events
-export class ContextChangeEventsArgs {
-    public constructor(
-        public readonly profileName: string | undefined,
-        public readonly accountId: string | undefined,
-        public readonly regions: string[]
```

Comment:
Not seeing any replacements for this, did this have a purpose? Is this tied to a different PR?

Reply:
The class became an interface. The regions were removed because the event is Credentials-centric, and regions was never used. We may want to consider having a separate region changed EventEmitter, but that isn't necessary for these changes.